Signal/src/util/OWSScreenLockUI.m

```diff
@@ -145,6 +145,9 @@ - (void)ensureScreenProtection
         shouldHaveScreenProtection,
         shouldHaveScreenLock,
         shouldShowBlockWindow);
+    if (self.screenBlockingWindow.hidden != shouldShowBlockWindow) {
```

Comment:
Presumably we want to log whenever it changes.

So since we assign the negated value, isn't this condition inverted?

Reply:
Fixed.